dht_bootstrap.go

```diff
@@ -10,8 +10,35 @@ import (
 	goprocess "github.com/jbenet/goprocess"
 	peer "github.com/libp2p/go-libp2p-peer"
 	routing "github.com/libp2p/go-libp2p-routing"
+	multiaddr "github.com/multiformats/go-multiaddr"
```

Comment:
nit: lo and behold, convention is to alias multiaddr as `ma`. Change would be appreciated for consistency.